File: engine/model/src/main/scala/net/server/channel/packet/cash/operation/MakePurchasePacket.scala
package net.server.channel.packet.cash.operation

class MakePurchasePacket(private var _action: Int, private var _useNX: Int, private var _snCS: Int) extends BaseCashOperationPacket(_action = _action) {
  def useNX: Int = _useNX

  def snCS: Int = _snCS
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/MakePurchasePacket.java
package net.server.channel.packet.cash.operation;

public class MakePurchasePacket extends BaseCashOperationPacket {
   private final Integer useNX;

   private final Integer snCS;

   public MakePurchasePacket(Integer action, Integer useNX, Integer snCS) {
      super(action);
      this.useNX = useNX;
      this.snCS = snCS;
   }

   public Integer useNX() {
      return useNX;
   }

   public Integer snCS() {
      return snCS;
   }
}
